rasa/nlu/training_data/message.py

```diff
@@ -11,6 +11,11 @@
 from rasa.nlu.utils import ordered
 
 
+INTENT_RANKING_KEY = "intent_ranking"
+INTENT_KEY = "intent"
+INTENT_CONFIDENCE_KEY = "confidence"
```

Comment:
it can be confidence of anything and let's move it to nlu/constants.py
```suggestion
CONFIDENCE = "confidence"
```